### ISSUE TITLE
Issues with lips color model 

### ISSUE THREAD

**Issue by ghost** [Kristina-Simakova/arfaces#1]
Great Work! Unfortunately the lips color change doesn't feel natural. How can I improve this ?
I have seen some sparkAR videos where the changing lips color filters look natural. I want to build an app which gives result closer to that.

**Comment by droid-girl** [comment]
@arsalyou  this tutorial was made just to demonstrate the principle of working with face texture. 
Here are the steps to improve the quality:
1. Take a reference face texture and make a better contour in Photoshop around the lips area. 
2. Fade the color with alpha value the closer you get to the contour, it will look more natural
3. To make sure that it works well with mouth open, make a transparent pixel thin line between the lips on your texture.
In conclusion: it is all about experimentation with face texture in Photoshop
